community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -36,32 +22,38 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Acquire lock
+    ansible.builtin.shell:
```

Comment:
If now shell functionality is used then the `command` module is prefered to `shell`. Ansible lint will error on this although it is temporarily disabled.